tests/unit_tests/blas/batch/syrk_batch_stride.cpp

```diff
@@ -63,14 +63,13 @@ int test(device *dev, oneapi::mkl::layout layout, int64_t batch_size) {
 
     upper_lower = (oneapi::mkl::uplo)(std::rand() % 2);
     if ((std::is_same<fp, float>::value) || (std::is_same<fp, double>::value)) {
-        trans = (oneapi::mkl::transpose)(std::rand() % 2);
+        trans = (std::rand() % 2) == 0 ? oneapi::mkl::transpose::nontrans
```

Comment:
I am confused by these changes. What exactly is the unsupported test condition (ie, what datatype/trans combination caused problems)?

The [docs](https://oneapi-src.github.io/oneMKL/domains/blas/syrk.html) say "Conjugation is never performed, even if trans = transpose::conjtrans" which is maybe strange but it's not the same as unsupported.

Reply:
mkl::transpose::conjtrans caused issue for complex precisions. It is not supported for complex precisions, hence removed from possible test conditions.